src/quic/node_quic_session.cc

```diff
@@ -563,26 +667,43 @@ void JSQuicSessionListener::OnHandshakeCompleted() {
   Context::Scope context_scope(env->context());
 
   QuicCryptoContext* ctx = session()->crypto_context();
+
+  QuicCallbackScope cb_scope(session());
+
   Local<Value> servername = Undefined(env->isolate());
+  Local<Value> validationErrorReason = v8::Null(env->isolate());
+  Local<Value> validationErrorCode = v8::Null(env->isolate());
```

Comment:
```suggestion
  Local<Value> validationErrorReason;
  Local<Value> validationErrorCode;
```
?

Reply:
Explicitly setting to `null` (as opposed to `undefined`) is intentional here. May revisit that later, but for now this should stay as is.